nextflow_schema.json

```
@@ -429,9 +442,16 @@
                         "unite-alleuk=8.3",
                         "unite-alleuk=8.2",
                         "unite-alleuk",
-                        "greengenes85"
+                        "greengenes85",
+                        "greengenes2",
+                        "greengenes2=2022.10"
                     ]
                 },
+                "qiime_ref_tax_custom": {
+                    "type": "string",
+                    "help_text": "Is preferred over `--qiime_ref_taxonomy`. A comma separated pair of (possibly gzipped) filepaths (sequence, taxonomy).",
```

```suggestion
                    "help_text": "Overwrites `--qiime_ref_taxonomy`. A comma separated pair of (possibly gzipped) filepaths (sequence, taxonomy).",
```

Maybe something like this, preferred seems to be the less buggy or better functioning parameter.

---

Also when I checked correctly, this parameter could be used instead of --qiime_ref_taxonomy. In line 19, only --qiime_ref_taxonomy is mentioned.

---

Actually the parameter description was a little lacking, I updated it in https://github.com/nf-core/ampliseq/pull/691